example/particle_leapfrog/parthinput.particle_leapfrog

```diff
@@ -45,10 +45,19 @@ nx2 = 8
 nx3 = 8
 
 <parthenon/time>
-tlim = 1.0
+tlim = 2.0
```

Comment:
Necessary change? `dt = 0.5` below instead?

Reply:
not a necessary change here. I changed it because I wanted to run an identical sim to the one in the regression test input deck for debugging.

Reply:
reverted.